src/fsharp/FSharp.Core/async.fsi

```diff
@@ -150,8 +223,25 @@ namespace Microsoft.FSharp.Control
         /// before being disposed.</returns>
         ///
         /// <category index="3">Cancellation and Exceptions</category>
-        /// 
-        /// <example-tbd></example-tbd>
+        ///
+        /// <example id="on-cancel-1">
+        /// <code lang="fsharp">
+        ///     [ 2; 3; 5; 7; 11 ]
+        /// |> List.iter
+        ///     (fun i ->
```

Comment:
You can format this on one line `|> List.iter (fun i ->` etc.   Same across all samples